deepchem/models/tensorgraph/layers.py

```diff
@@ -114,6 +114,46 @@ def _record_variable_scope(self, local_scope):
     else:
       self.variable_scope = local_scope
 
+  def set_summary(self,
+                  summary_op,
+                  name=None,
+                  summary_description=None,
+                  collections=None):
+    """Annotates a tensor with a tf.summary operation
+    Collects data from self.out_tensor by default but can be changed by setting self.tb_input to another tensor in create_tensor
+
+
+    Parameters
+    ----------
+    summary_op: summary operation to annotate node
```

Comment:
There are supposed to be numpy docstrings
https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt

``` python
variable_name: type
     variable description
```